cmd/core-command/Dockerfile

```diff
@@ -49,5 +51,13 @@ WORKDIR /
 COPY --from=builder /edgex-go/cmd/core-command/Attribution.txt /
 COPY --from=builder /edgex-go/cmd/core-command/core-command /
 COPY --from=builder /edgex-go/cmd/core-command/res/configuration.toml /res/configuration.toml
-ENTRYPOINT ["/core-command"]
-CMD ["-cp=consul.http://edgex-core-consul:8500", "--registry", "--confdir=/res"]
+
+ENV CONSUL_HOST=edgex-core-consul
+ENV CONSUL_PORT=8500
+
+# setup entrypoint script
+COPY --from=builder /edgex-go/cmd/core-command/entrypoint.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/entrypoint.sh \
+    && ln -s /usr/local/bin/entrypoint.sh /
+
+ENTRYPOINT ["entrypoint.sh"]
```

Comment:
Change to  `ENTRYPOINT ["/entrypoint.sh"]`

Also keep CMD and reference it from entry point script as `/core-command $@` so it can still be overridden in compose file. Same for other Docker files.